### ISSUE TITLE
[MAS4.2.1][Screen reader-How to Build a Bot] In voiceover mode, the contents are announced as “1 of 2 plan “ and "2 of 2 review the bot".

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1900]
**Actual Result:**
When the user navigates through contents available under the heading "How to build a bot", the voiceover announces the content "Plan" as “1 of 2 plan“ and "2 of 2 review the bot" which might confuse the user.

**Expected Result:**
The voiceover should read as “plan“ only instead of "1 of 2 Plan".
For remaining elements also Build, Test, Publish, Connect, Evaluate screen reader should read the proper info.

**Note:**
Same issue observed for Build, Test, Publish, Connect, Evaluate.
It is also observed, that the voiceover announces the "Plan" twice as "Plan 1 of 2" and "Plan". It should be announce only once. (Please refer the attached video for the reference of both the issue's)

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "How to build a Bot" section.
4. Observe the issue.

**User Impact:**
Users who rely on screen reader will be affected as this will confuse them to navigate the control if voiceover announces unnecessary information.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}](url)

**Attachment:**
[Contents are announced as “1 of 1 plan “ and "2 of 2 review the bot".zip](https://github.com/microsoft/BotFramework-Emulator/files/3686334/Contents.are.announced.as.1.of.1.plan.and.2.of.2.review.the.bot.zip)

**Comment by ShikhaMishra11** [comment]
Verified ,issue is fixed now so closing the bug.
